oshmem/mca/sshmem/verbs/sshmem_verbs_component.c

```diff
@@ -100,6 +101,11 @@ verbs_runtime_query(mca_base_module_t **module,
     *priority = 0;
     *module = NULL;
 
+    /* If fork support is requested, try to enable it */
+    if (OSHMEM_SUCCESS != (rc = opal_common_verbs_fork_test())) {
+        return OSHMEM_ERROR;
```

Comment:
Sure, makes sense.

The common verbs code will opal_show_help(), which is good.  The oob ud component calls the opal_output_verbose(), which can show the result even if the default MCA var value was -1.  As such, it's outputting extra / verbose information, which seems helpful in most (all?) cases.  But I guess this is just my opinion.  :-)